src/NUnitTestAdapter/TestConverter.cs

```diff
@@ -159,14 +159,24 @@ public DiaNavigationData GetNavigationData(string className, string methodName)
         {
             if (this.DiaSession == null) return null;
 
+            // First try using the class and method names provided directly
             var navData = DiaSession.GetNavigationData(className, methodName);
```

Comment:
If `NavigationDataHelper` is not `null` and the method is defined in a base class this statement will always return `null` navigation data and later flow will proceed through if `(NavigationDataHelper != null)` anyway; I would suggest that you try with the provided class name only when `NavigationDataHelper` is `null`. I'm assuming that `NavigationDataHelper.GetDefiningClassName` call isn't so expensive anyway(unless you know otherwise).